readthedocs/api/v3/serializers.py

```diff
@@ -594,6 +627,9 @@ def get_subproject_of(self, obj):
             return None
 
 
+# FIXME: this override isn't needed, but tests will fail if removed.
+# We may have been relying on a weird behavior of using this class
+# as a base class of another.
```

Comment:
This wrong behavior is described in https://github.com/readthedocs/readthedocs.org/issues/8394